doc/reference/misc.md

```diff
@@ -1790,7 +1790,8 @@ Please document me!
 :::
 
 In the copy of str replace the string old with string new.
-Count controls the number of replacements (count < 0 equals to #f).
+Count controls the number of replacements (#f means no limit).
+The procedure returns the input string if count is #t or <= 0.
```

Comment:
you shouldn't have semantics at all for `#t`. Acceptable value is a fixnum or `#f`